src/react/components/searchNoResults.js

```diff
@@ -0,0 +1,61 @@
+import HeroIcon from "./heroIcon";
+import React from "react";
+
+const columns = [
+  {
+    heading: 'Popular Topics',
+    links: [
+      {title: 'Reunion 2021', url: 'https://reunion.stanford.edu/'},
+      {title: 'Programs & Resources for Students', url: 'https://students.alumni.stanford.edu/'},
+      {title: 'Request your official Stanford transcript', url: 'https://registrar.stanford.edu/students/transcripts/transcripts-alumni'},
+      {title: 'Stanford Alumni Email', url: 'https://alumni.stanford.edu/perks/email/'},
+      {title: 'Alumni Membership', url: 'https://alumni.stanford.edu/membership/'},
+    ]
+  },
+  {
+    heading: 'Contact Information',
+    links: [
+      {title: 'SAA Customer Service', url: 'https://alumni.stanford.edu/goto/snowcs'},
+      {title: 'Stanford University Contact Information', url: 'https://www.stanford.edu/contact/'},
+      {title: 'SAA Departments', url: 'https://alumni.stanford.edu/about/departments/'},
+    ]
+  }
+]
+
+const listClasses = "algolia-search--no-results-list list-none pl-0 children:leading-none";
+const listItemClasses = "algolia-search--no-results-list-item block text-left mb-6";
+const linkClasses = "algolia-search--no-results-link flex w-fit transition-colors no-underline underline-offset hover:underline focus:underline";
+
+const SearchNoResults = ({ heading, body }) => (
+  <div className="algolia-search--no-results">
+    <h2 className="algolia-search--no-results-heading">
+      {heading}
+    </h2>
+    <p className="algolia-search--no-results-body">{body}</p>
+    <div>
+      <h3 className="algolia-search--no-results-suggestions-heading mt-40 mb-12 md:mb-20">Consider Browsing by Category:</h3>
+      <div className="w-full grid">
+        <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-2 gap-y-16 md:gap-x-8 xl:gap-x-12 justify-items-start grid-gap items-start">
```

Comment:
I see...so this repo isn't using TailwindCSS in the same way as our Storyblok sites. I think this grid-gap class (a custom TW class from Decanter v7) isn't doing anything here and everywhere that uses it in this repo. For example, there is no gap between the filter and the search results region
![Screen Shot 2021-10-12 at 4 11 38 PM](https://user-images.githubusercontent.com/42749717/137041153-e79150c9-335b-45f9-a467-650bb3f512e2.png)
: